server/src/com/cloud/storage/StorageManagerImpl.java

```diff
@@ -1736,7 +1737,10 @@ public boolean storagePoolHasEnoughSpace(List<Volume> volumes, StoragePool pool)
                     allocatedSizeWithtemplate = _capacityMgr.getAllocatedPoolCapacity(poolVO, tmpl);
                 }
             }
-            if (volumeVO.getState() != Volume.State.Ready) {
+            // A ready state volume is already allocated in a pool. so the asking size is zero for it.
+            // In case the volume is moving across pools or is not ready yet, the asking size has to be computed
+            s_logger.debug("pool id for the volume with id: " + volumeVO.getId() + " is: " + volumeVO.getPoolId());
```

Comment:
I guess there is no consensus or guideline around logging ids in debug logs (uuids vs ids) though I've seen most logs messages using IDs instead of UUIDs (perhaps because most code relied on IDs than UUIDs, as UUIDs were introduced only in that last 2 years or so).

Though the logged message is not verbose enough wrt what it was doing, may be you can add some context to say what it is doing etc?